### DISCUSSION TITLE
How many 🍌 are in pyjamas in the kids 🎶?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1075] [Galaxy Trivia] [answered]
**Bonus:** What is the collective term for a group of ... ?
![LionKingHappyGIF](https://github.com/ParanoidUser/yolo/assets/5120290/d9a5e911-cf7b-471c-8ed6-17a354e24d06)

**Comment by peterbaccaro** [comment, marked as answer]
There are **two** banana in pyjamas named B1 and B2.
The main collective nouns for Hyenas are a **cackle**, a **clan**, and a **pack**.

**Reply by NomanGhiasy** [reply]
You are correct
